core/unpack/unpacker_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package unpack
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/opencontainers/go-digest"
+	"github.com/opencontainers/image-spec/identity"
+)
+
+func generateRandomDiffIDs(num int) []digest.Digest {
+	diffIDs := make([]digest.Digest, num)
+	for i := 0; i < num; i++ {
+		diffIDs[i] = digest.FromString(fmt.Sprintf("sha256:%d", i))
```

Comment:
This doesn't seem to be "random" nor a valid diff ID

Reply:
Hi @AkihiroSuda yes this is more like generating N valid digest so we can benchmark `identity.ChainID` and  `identity.ChainIDs`.

I updated to use a diffID set from a golang image, and generate N valid DiffIDs by appending those golang diffIDs. The benchmark results look the same. Let me know if this makes sense, or if there is better way to construct valid random diffIDs.

thanks!